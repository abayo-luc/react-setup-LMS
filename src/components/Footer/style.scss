.footer {
  background-color: #007bff;
  color: #fff;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding: 25px 15px;
  bottom: 0px;
  position: fixed;
  width: 100%;
  .contact {
    ul {
      list-style: none;
      li {
        padding-top: 5px;
      }
    }
  }
  .copyright {
    text-align: center;
  }
  h3 {
    font-size: 18px;
  }
  p {
    font-size: 12px;
  }
}
