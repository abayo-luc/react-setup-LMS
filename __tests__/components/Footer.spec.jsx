import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../src/components/Footer/Footer';

const jSonWarper = toJson(shallow(<Footer />));

describe('Footer Component', () => {
  test('should match the snapshot', () => {
    expect(jSonWarper).toMatchSnapshot();
  });
});
