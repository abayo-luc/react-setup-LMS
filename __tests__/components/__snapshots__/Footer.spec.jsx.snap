// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should match the snapshot 1`] = `
<div
  className="footer"
>
  <div
    className="contact"
  >
    <ul>
      <li>
        Phone: +250789277275
      </li>
      <li>
        Email: info@readit.info
      </li>
      <li>
        Address: KG 15 Av
      </li>
    </ul>
  </div>
  <div
    className="copyright"
  >
    <h3>
      © 2017 ReadIt. All rights reserved | Design by Titans Devs
    </h3>
    <p>
      “Save the World by Sharing a Part of Your Brain!”
    </p>
  </div>
  <div
    className="links"
  >
    <ul>
      <ul>
        Help
      </ul>
      <ul>
        Privacy
      </ul>
      <ul>
        Terms
      </ul>
      <ul>
        Careers
      </ul>
    </ul>
  </div>
</div>
`;
